### PR TITLE
Added missing -b argument in the contributing.md readme file

### PR DESCRIPTION
**Pull request by claudiobernasconi**
Simple CONTRIBUTING.MD fix.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ While you're working away in your branch it's quite possible that your upstream/
 1. [Stash](http://progit.org/book/ch6-3.html) any un-committed changes you need to
 1. `git checkout master`
 1. `git pull upstream master`
-1. `git checkout myBranch`
+1. `git checkout -b myBranch`
```
Comment (thomaslevesque) [inline CONTRIBUTING.md]: `git checkout -b` creates a branch, but in this case the branch already exists. This section only describes how to rebase it on top of `master`.

**Comment by thomaslevesque** [comment]
The current instructions seem correct to me (this part at least). Thanks anyway, your efforts to improve the doc are appreciated!

**Comment by blairconrad** [comment]
@claudiobernasconi sorry you went to the trouble. When we were chatting I hadn't actually gone to the doc and read the full text. Otherwise I (I hope) would've realized that the change wasn't necessary.
🙇

**Comment by claudiobernasconi** [comment]
@thomaslevesque @blairconrad no problem. As promised in the chat I went over the doc again today. I found some little improvments I thinkt that are worth adding. I added a commit to the branch on my fork. Maybe you could reopen this PR so that I can add those changes? Or should I open a new PR using a new feature branch?

**Comment by adamralph** [comment]
@claudiobernasconi 

> Or should I open a new PR using a new feature branch?

Yes please, that would be better. Please ensure it only contains the new commit and not the commit which adds the `-b`.

> Also I found that in chapter Resharper Artifacts we are saying that "A definitive Resharper settings file is in the pipeline.". Is there a ReSharper file in the meantime? If not, maybe we should either remove that sentense or open a issue so that it gets done in the future?

I believe we should remove that sentence. We are not planning to add a R# settings file.

**Comment by claudiobernasconi** [comment]
@adamralph thanks for helping out. I will do as you suggested and I will removed the metioned sentense as well. This OR can be closed and I will open-up a new one once its completed. Thank you.

**Pull request closed without merging**